### ISSUE TITLE
Conformation to WCAG 2.1

### ISSUE THREAD

**Issue by WebSpider** [filesender/filesender#1145]
Every since the directive or the European Union about Web Accessibility became into effect (https://eur-lex.europa.eu/eli/dir/2016/2102/oj), we have seen increasing requests for all services at SURF to start complying to these.

The local government is formulating timelines under 2y for all commercial and non-commercial sites to comply to these directives.

For our instance of filesender this boils down to WCAG 2.1 compliance (https://www.w3.org/TR/WCAG21/), as complete as possible, whenever applicable.

As SURF we'd like to suggest to start addressing this issue, especially since we know this will become an issue for more and more service providers (at least those based in the EU) in the foreseeable future.

**Comment by monkeyiq** [comment]
Given the time frame it seems reasonable to target FileSender 3.x for this update.

**Comment by monkeyiq** [comment]
I will suggest this to the FileSender board for the next MoU which IIRC should be discussed around the end of 2022.

**Comment by monkeyiq** [comment]
I am not sure where this fits with the potential 2023 UI rewrite.

**Comment by WebSpider** [comment]
Thanks for pinging this @monkeyiq, I will put this on the agenda for the UI/UX preparations.

**Comment by monkeyiq** [comment]
Sounds good!

**Comment by wnas** [comment]
I would like to see this upgraded to 2.2 as this is, or will be, the european standard.
